features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/logic/flot.app-by-popularity-over-time.js

```diff
@@ -23,7 +23,7 @@ function drawGraphs() {
                    'endDate': to
                },
                success: function (response) {
-                   parsedResponse = JSON.parse(response);
+                   parsedResponse = response;
                    drawPopularityOverTime(parsedResponse);
```

Comment:
variable `parsedResponse` is used in [line 38](https://github.com/wso2/carbon-appmgt/pull/146/files#diff-57cb2c4858a29cd95339f0512e93d071R38).
Fixed: variable is renamed and reduced its scope to function-local, see commit 1553f4bd70a29ad5ac7831b2a122b3bc92f15f6e of PR #240